### ISSUE TITLE
A11y_AzureDataStudioSQLMigration_MigratetoAzureSQL_Databasebackup-networkshare_ColorContrast:The tooltip provided for resource group combo box has a color contrast of 2.8:1 which is less than 3:1 criteria.

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/azuredatastudio#15767]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yWCAG2.1;#A11yTCS;#A11ySev2;#SH_AZUREDATASTUDIOSQLMigration_June2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.4.11;#AINotInScope;#Colorcontrast;#A11yMAS

### Environment Details:
Application Name: SQL Azure Data Studio
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure SQL Migration.
2. Connect to any sql server.
3. Tab till Azure SQL migration Tab and press enter.
4. Tab till Migrate to azure sql and press enter.
5. Add azure account and tab till next button and press enter.
6. Select SQL server on azure virtual machine radio button and fill all the required fields and tab till next button and press enter.
7. Check Online migration radio button and tab till next button and press enter.
8. Check "My database are on a network share" radio button.
9. Tab till resource group combo box.
10. Open Color contrast analyser and check the color contrast for tooltip.

### Actual:
The tooltip provided for resource group combo box has a color contrast of 2.8:1 which is less than 3:1 criteria.

### Expected:
The tooltip should meet the minimum color contrast ratio that is should be greater or equal to 3:1.

### User Impact:
People with low vision often have difficulty in using the combo box.

**Comment by aasimkhan30** [comment]
The tooltip does not exist for the editable dropdown. However, the dropdown options have tooltips with the same bug.

**Comment by alanrenmsft** [comment]
![image](https://user-images.githubusercontent.com/13777222/123876931-15d60e80-d8f1-11eb-8de5-61f5bf6dfa90.png)


![image](https://user-images.githubusercontent.com/13777222/123876677-96e0d600-d8f0-11eb-9276-b5fab9050fe6.png)

@Shivamd1 I just checked, the contrast ratio is 9.4:1, see attached screenshots. not sure why are you seeing 2.8:1. As per Guy Barker, you should use the most contrast part of the text and background not the edges.

**Comment by Shivamd1** [comment]
@alanrenmsft The issue is for the tooltip which is provided for resource group and storage account combo box, the color contrast for which is below 3:1.

**Comment by alanrenmsft** [comment]
@Shivamd1 please attach screenshots

**Comment by alanrenmsft** [comment]
had a chat with Akhil from the test team, they actually meant to say the icon's contrast ratio, not the tooltip.

![image](https://user-images.githubusercontent.com/13777222/124167274-b6930e00-da58-11eb-80cc-ece3efc0aace.png)

**Comment by alanrenmsft** [comment]
the icon is a font icon, and if you check its css style, the color is set to RGB(74,74,74) and font-weight: 400 (normal)
![image](https://user-images.githubusercontent.com/13777222/124200826-23270080-da8b-11eb-8cd7-e1fcab0d1a4f.png)

the contrast ratio is actually good:
![image](https://user-images.githubusercontent.com/13777222/124200671-c9bed180-da8a-11eb-9e93-e1ee0d6a6157.png)

but when you use color contrast analyzer, we are seeing this:
![image](https://user-images.githubusercontent.com/13777222/124200930-6bdeb980-da8b-11eb-8dd6-cc73898775b5.png)

this is common even for characters, the three characters in the screenshot below are 'ows'
![image](https://user-images.githubusercontent.com/13777222/124201013-b06a5500-da8b-11eb-8ffe-2a037c84b098.png)

for straight lines, the color is usually the actual specified color, when there are curves, the color will be lighter than specified. I will ask the accessibility expert to see what should we do about this.

**Comment by alanrenmsft** [comment]
as per the accessibility SME, it is recommended to increase the thickness of this particular icon to make it more contrast.

**Comment by fsteffi** [comment]
issue does not repro
#Closed;

![15767](https://user-images.githubusercontent.com/24580570/228226947-25ad0157-911d-473d-9c00-0bb85ad0ac79.png)
Version: 1.43.0-insider (user setup)
Commit: 9d16a48dee06c0533c5c6e405e775685d098a5ae
Date: 2023-03-20T05:49:59.560Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621
